apps/pie-storybook/stories/testing/pie-chip.test.stories.ts

```diff
@@ -0,0 +1,84 @@
+import { html, nothing } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { action } from '@storybook/addon-actions';
+import '@justeattakeaway/pie-chip';
+import '@justeattakeaway/pie-icons-webc/dist/IconHeartFilled.js';
+
+import { type ChipProps as ChipPropsBase, variants, defaultProps } from '@justeattakeaway/pie-chip';
+import { type Meta } from '@storybook/web-components';
+import { type SlottedComponentProps } from '../../types';
+
+import {
+    createStory, createVariantStory, type TemplateFunction, sanitizeAndRenderHTML,
+} from '../../utilities';
+
+type ChipProps = Omit<SlottedComponentProps<ChipPropsBase>, 'aria'> & { showIcon: boolean };
+type ChipStoryMeta = Meta<ChipProps>;
+
+const defaultArgs: ChipProps = {
+    ...defaultProps,
+    showIcon: false,
+    slot: 'String',
+};
+
+const chipStoryMeta: ChipStoryMeta & { showInTestingDeployment?: boolean } = {
+    title: 'Chip',
+    showInTestingDeployment: true,
+    component: 'pie-chip'
+};
+
+export default chipStoryMeta;
+
+const clickAction = action('clicked');
+const closeAction = action('pie-chip-close');
+
+const Template: TemplateFunction<ChipProps> = ({
```

Comment:
There's obviously a bit of duplication regarding the template both here and in the main story file. I spoke to @jamieomaguire about possibly moving these to a seperate file an exporting so it can be reused, but this solution is no worse than what we had before with the template in the spec file 👍🏼